clang_tools/install.py

```diff
@@ -38,7 +38,7 @@ def is_installed(tool_name: str, version: str) -> Optional[Path]:
     )
     try:
         result = subprocess.run(
-            [exe_name, "--version"], capture_output=True, check=True
+            [exe_name, "--version"], stdout=subprocess.PIPE, check=True
```

Comment:
This isn't totally equivalent, but I think it will work in this case.

According to the docs:
> If capture_output is true, `stdout` and `stderr` will be captured. When used, the internal [Popen](https://docs.python.org/3/library/subprocess.html#subprocess.Popen) object is automatically created with `stdout=PIPE` and `stderr=PIPE`. The `stdout` and `stderr` arguments may not be supplied at the same time as capture_output. If you wish to capture and combine both streams into one, use `stdout=PIPE` and `stderr=STDOUT` instead of capture_output.

Meaning, if you want to apply this change to cpp-linter pkg's use of `subprocess.run()`, then you'de have to use
```py
result = subprocess.run(cmds, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
```
because we actually use the stderr output in the cpp-linter pkg.